lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3738,7 +3738,11 @@ namespace {
         assert(((decl->getNumParams() == argNames.size() + 1) || isAccessor) &&
                (*selfIdx < decl->getNumParams()) && "where's self?");
       } else {
-        assert(decl->getNumParams() == argNames.size() || isAccessor);
+        // Variadic members, such as "void test(...)" will appear to have no
+        // parameters; that's OK.
```

Comment:
This is a bit confusing to me, I'd prefer reading something in the sense

> variadic parameter is not counted in decl->getNumParams, for example function that only accepts a variadic parameter will return 0 from getNumParams

or just dropping the comment altogether, the code speaks for itself well enough.

Reply:
I think you're right, the code speaks for itself. I'll drop the comment.